spring-cloud-dataflow-server-core/src/main/java/org/springframework/cloud/dataflow/server/service/TaskDeleteService.java

```diff
@@ -29,6 +32,13 @@
 	 */
 	void cleanupExecution(long id);
 
+	/**
+	 * Delete one or more Task executions.
+	 *
+	 * @param id the execution id
+	 */
+	void deleteOneOrMoreTaskExecutions(Set<Long> ids);
```

Comment:
Why not deleteTaskExecutions instead of deleteOneOrMoreTaskExecutions.

Reply:
Changed.